gemsfx/src/main/resources/com/dlsc/gemsfx/enhanced-password-field.css

```diff
@@ -5,4 +5,12 @@
 
 .enhanced-password-field .right-icon-wrapper .ikonli-font-icon {
     -fx-icon-size: 15px;
+}
+
+.enhanced-password-field:showing .right-icon-wrapper .ikonli-font-icon {
+    -fx-icon-code: mdi-eye-off;
```

Comment:
can you please use SVG instead of FontIcon? Libraries ideally use SVG, apps are fine using FontIcon.